src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figuresList = new Figure[6];
+
+        for (int i = 0; i < figuresList.length; i++) {
+            if (i < 3) {
```

Comment:
depend on array length, don't hardcode, if you decide to have a bigger array, you will need to make changes in two places instead of one